### ISSUE TITLE
Build failure with optparse-applicative 0.13

### ISSUE THREAD

**Issue by snoyberg** [GregorySchwartz/find-clumpiness#1]
```
[1 of 1] Compiling Main             ( app/Main.hs, dist/build/find-clumpiness/find-clumpiness-tmp/Main.o )

app/Main.hs:42:10: error:
    • Variable not in scope: (<>) :: Mod f0 a0 -> Mod f1 a1 -> t1
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Options.Applicative),
        ‘<*>’ (imported from Options.Applicative),
        ‘*>’ (imported from Options.Applicative)

app/Main.hs:43:10: error:
    • Variable not in scope: (<>) :: t1 -> Mod f2 a2 -> t0
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Options.Applicative),
        ‘<*>’ (imported from Options.Applicative),
        ‘*>’ (imported from Options.Applicative)

app/Main.hs:44:10: error:
    • Variable not in scope:
        (<>) :: t0 -> Mod f3 a3 -> Mod OptionFields String
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Options.Applicative),
        ‘<*>’ (imported from Options.Applicative),
```

**Comment by GregorySchwartz** [comment]
Waiting on https://github.com/choener/ForestStructures/issues/1 and https://github.com/choener/BiobaseNewick/issues/1.
